distribution/pom.xml

```diff
@@ -178,12 +178,37 @@
                                 </goals>
                                 <configuration>
                                     <finalName>apache-druid-${project.parent.version}</finalName>
-                                    <tarLongFileMode>posix</tarLongFileMode>
+                                    <tarLongFileMode>gnu</tarLongFileMode>
```

Comment:
What does this change do?

Reply:
I believe it controls the archive format used, as described here: https://www.gnu.org/software/tar/manual/html_section/tar_67.html#Formats

I was attempting to fix the PaxHeader files generated when Julian tried extracting the tarball as described here: https://lists.apache.org/thread.html/97bdf5827c0a22b0174c0e9e48df42c513edfd477ed8a65518485960@%3Cdev.druid.apache.org%3E

Having said that, I'm now inclined to go back to using posix, primarily because it seems to be the standard archive format used by Apache projects, since that is how the assembly plugin in the Apache parent POM is configured: https://github.com/apache/maven-apache-parent/blob/apache-21/pom.xml#L374